tests/props.spec.ts

```diff
@@ -82,4 +83,32 @@ describe('props', () => {
       object: {}
     })
   })
+
+  it('should return the updated props on 2 way binding', async () => {
```

Comment:
This file is probbly the wrong place for this test, please advise where it would be best.

Reply:
Here is probably fine. Probably could re-think the test layout at some point.